docs/getting_started/what_is_GTFS.md

```diff
@@ -1,13 +1,30 @@
+The General Transit Feed Specification, also known as GTFS, is a standardized data format that provides a structure for public transit agencies to describe the details of their services such as schedules, stops, fares, etc.
 
-GTFS consists of two main parts: GTFS Schedule and GTFS Realtime. GTFS Schedule contains information about routes, schedules, fares, and geographic transit details, and it is presented in simple text files. This straightforward format allows for easy creation and maintenance without relying on complex or proprietary software.
+It allows public transit agencies to publish their transit data in a format that can be consumed by a wide variety of software applications, most commonly trip planners. This means that users can easily get travel information to access public transit services by using their smartphones or similar devices.
 
-GTFS Realtime contains trip updates, vehicle positions, and service alerts. It is based on Protocol Buffers, which are a language (and platform) neutral mechanism for serializing structured data.
+<img class="center" width="560" height="100%" src="../../../assets/what_is_gtfs_001.png">
 
-GTFS is supported around the world and its use, importance, and scope has been increasing. It’s likely that an agency you know already uses GTFS to represent routes, schedule, stop locations, and other information, and that riders are already consuming it via various applications.
+Today, GTFS is the go-to [Open Standard](https://www.interoperablemobility.org/definitions/#open_standard) for thousands of public transport providers worldwide. Some agencies produce this data themselves, while others employ a vendor to create and maintain data for them. 
+
+## Two specifications in one
+
+GTFS consists of two main parts: [GTFS Schedule](/documentation/schedule/reference) and [GTFS Realtime](/documentation/realtime/reference).
+
+<img class="center" width="560" height="100%" src="../../../assets/what_is_gtfs_002.png">
+
+GTFS Schedule contains information about routes, schedules, fares, and geographic transit details among many other features, and it is presented in simple text files[^1]. This straightforward format allows for easy creation and maintenance without relying on complex or proprietary software.
+
+GTFS Realtime contains trip updates, vehicle positions, and service alerts, using a different format based on [Protocol Buffers](https://developers.google.com/protocol-buffers/). This part of GTFS works in conjunction with GTFS Schedule in order to inform passengers of service disruptions and updated arrival times.
+
+The detailed information for both specifications is hosted in documents known as the Reference documents. An updated copy of this information can be found in the [Technical Documentation section](/documentation/overview).
```

Comment:
I think the term "Reference" under the "Technical documentation" section speaks for itself. I wouldn't call this a copy.

```suggestion
The GTFS Schedule and GTFS Realtime reference documentation is available in the [Technical Documentation section](/documentation/overview).
```